inputs/mvs_config.json

```diff
@@ -331,17 +287,17 @@
             "file_name": "pv_gen_merra2_2014_eff1_tilt40_az180.csv",
             "installedCap": {
                 "unit": "kWp",
-                "value": 0
-            },
-            "maximumCap": {
-                "unit": "kWp",
-                "value": 1000
+                "value": 1500
             },
             "label": "PV plant (mono)",
             "lifetime": {
                 "unit": "year",
                 "value": 30
             },
+            "maximumCap": {
```

Comment:
I would go for no limitation of maximumCap = no contraint; again for simplicity

Reply:
@SabineHaas , @smartie2076 - how should I change maximumCap in `energyProduction.csv` so that there is no constraint ?

Reply:
@Piranias is the use of ´maximumCap´ documented somewhere in the readthedocs? I did not figure out the valid values it can have, ie. float and "None", afaik.